go/vt/dbconfigs/dbconfigs.go

```diff
@@ -69,14 +69,15 @@ const (
 	// AllPrivs user should have more privileges than App (should include possibility to do
 	// schema changes and write to internal Vitess tables), but it shouldn't have SUPER
 	// privilege like Dba has.
-	AllPrivs = "allprivs"
-	Dba      = "dba"
-	Filtered = "filtered"
-	Repl     = "repl"
+	AllPrivs     = "allprivs"
+	Dba          = "dba"
+	Filtered     = "filtered"
+	Repl         = "repl"
+	ExternalRepl = "erepl"
```

Comment:
I see that you decided to go the hard-coded way :).

Reply:
Yeah, I'm going to add here what we discussed in slack so it stays documented for posterity.

At the moment, existent framework makes hard to add custom users. It also requires that added users are defined via flags, which does not allow us to add users dynamically. 

We discussed that we should do a more general refactor of how mysql configs are handled and revamp the whole dbconfigs module. That should be done outside of the scope of this PR. For now we will:
1. Add external_mysql that only provides a username. VReplication will always set this external user to be "erepl"
2. erepl will be defined in dbconfigs as any other user.
3. Future refactor of dbconfigs will allow us to support custom users in a more flexible way